modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -32,6 +32,7 @@ import {ModalEditFolder} from './ModalEditFolder';
 import {deleteObjectDefinition, getFolderActions} from './objectDefinitionUtil';
 
 import './ViewObjectDefinitions.scss';
+import {ModalDeletionNotAllowed} from '../ModalDeletionNotAllowed';
```

Comment:
@danvb112 I'm not sure, but I believe we should sort this import as well, what do you think?